post_install.sh

```diff
@@ -1,11 +1,26 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-chmod +x /usr/local/movienight/MovieNight
+## Who will run the jail's primary service
+service_port="8089"        # service_port == UID
```

Comment:
Why using the listening port as UID ?

Reply:
Typically, the Username is similar to the name of the primary service and the User ID usually matches the default port being used.

Reply:
Ok, but, does it have any technical interest or is this just a way to remind what is the listening port of the jail ?

Reply:
It's just a _suggested_ good practice... I read it about in the old FreeNAS documentation awhile back. I'll share the link if I can find it it. I follow this practice for all of my plugins bur it is certainly not required. Feel free to change it if you want

Reply:
Ok, that's what I has assumed. Thank you ! Thank a lot for your precious help !

Reply:
I'm glad I could help :slightly_smiling_face: